declaration/AxisManager.d.ts

```diff
@@ -0,0 +1,26 @@
+import { AxesOption } from "./Axes";
+export interface Axis {
+    [key: string]: number;
+}
+export interface AxisOption {
```

Comment:
`declaration` directory is only definition file.
`src` directory is only source.